src/main/webapp/WEB-INF/views/driver/newdrv.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
```

Comment:
change the name of jsp files, use full words
newdrv -> addDriver

Reply:
Done